packages/web-components/fast-components/src/color/utilities/color-constants.ts

```diff
@@ -13,11 +13,9 @@ export const black = SwatchRGB.create(0, 0, 0);
 /**
  * @internal
  */
-export const middleGrey = SwatchRGB.create(0.5, 0.5, 0.5);
+export const middleGrey = SwatchRGB.from(parseColorHexRGB("#808080")!);
```

Comment:
does this change result in a different gray?

Reply:
It does change things ever so slightly. It parses as 0.501... iirc, which affects some of the calculations. I believe we made this change in another place before, but this makes the ramp exactly the same the first time and when you go back and click on the 808080 swatch.

Reply:
bummer, okay